documentation/package.json

```diff
@@ -1,18 +1,23 @@
 {
-  "name": "documentation",
+  "name": "identity",
```

Comment:
Same thing with being unique here. We probably need to rename this to `identity-dev` for example. Because otherwise we have multiple configs with the same id in the wiki and yarn doesn't like that.
I hopes that just changing the version would solve that, but it expects a different name sadly.
Maybe someone here has a better solution